docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
May be call them `Daemons` (unix daemon analogy) as it is somehow a background process, not interactive, and can respond (but not change) certain actions (system events).